src/_main_/utils/common.py

```diff
@@ -153,7 +153,7 @@ def parse_string(s):
 
 def parse_int(b):
     if not str(b).isdigit():
-        raise ValueError("Input must be a digit")
+        return None
```

Comment:
This is strange.  If we return None then do we check if its None in the validator?

Reply:
no

Reply:
I have added that now